stateoftheuniverse/widgets/moon_functions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+"""
+A function to return moon phase & moon brightness.
+"""
+
+import numpy as np
+import astroplan
+import astropy
+from astropy.time import Time
+import matplotlib.pyplot as plt
+from astropy.coordinates import get_moon, get_sun
+import datetime
+import warnings
+warnings.filterwarnings('ignore')
+
+def get_moon_illumination(time: datetime) -> float:
+    """ A function to return the moon illumination
+
+    I'll fill this in later
+    """
+
+    astropytime = Time(time)
+    try:
+        illumination = astroplan.moon.moon_illumination(astropytime)
+    except:
```

Comment:
Bare `except` clauses are generally a bad idea: _"When catching exceptions, mention specific exceptions whenever possible instead of using a bare `except:` clause:_" ([PEP 8](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)).

Also, I think in this case we don't even need a `try/except` block. If something breaks, let it break; the resulting exception and stack traceback will likely be more useful than just seeing this error message :)

Reply:
Don't we want to include try excepts so that output is returned even if one of the widgets fails?

Reply:
I'm going to incorporate the same try/except setup as in #8, for now.